prewl/prompts.py

```diff
@@ -48,7 +50,7 @@ def complete(self, prompt_config_entry, include_output=False):
 
         """
         if include_output:
-            assert set(self.inputs + [self.output]) == set(prompt_config_entry.keys()), "Inputs and output don't match the keys provided"
+            # assert set(self.inputs + [self.output]) == set(prompt_config_entry.keys()), "Inputs and output don't match the keys provided"
```

Comment:
@jlaframboise Missed one?
```suggestion
            assert set(self.inputs + [self.output]) == set(prompt_config_entry.keys()), "Inputs and output don't match the keys provided"
```

Reply:
@jlaframboise I think this is the only one that remains in the way from merging...just want to be sure that this is one to be modified like the otheres.